### ISSUE TITLE
Three questions and suggests

### ISSUE THREAD

**Issue by ttgump** [shahsohil/DCC#3]
I have some questions about the DCC method:
1. I learned that in nowadays because of the use of dropout and ReLU, the layer-wise pretraining of autoencoder is not necessary (See the ReLU paper). If layer-wise pretraining can be skipped, it can save lots of time.
2. Denoise autoencoder can improve the performance of clustering. The DCC model used dropout as denoising layers. But for some numerical data, in my experiences, such as the protein data in the RCC paper, it is better to add the Gaussian noises that can help the performance. So why don't use both of Gaussian noise and dropout?
3. How to extract the learned clusters by DCC. I mean, after training DCC model, how can I extract the cluster assignment of each sample? 
Thanks!

**Comment by shahsohil** [comment]
Thanks Tian. Those are really good suggestion. Worth exploring them using DCC model. 

I just added support for extracting cluster assignment. Now you should be able to load them from features.mat file under your results directory.
